src/messages/components/CMessageRow.vue

```diff
@@ -97,7 +102,10 @@ export default {
       return `${diff.hours}:${diff.minutes}:${diff.seconds}`;
     },
     canRequeue() {
-      return ['Success', 'Failure', 'Canceled', 'Skipped'].includes(this.message.status);
+      return (
+        ['Success', 'Failure', 'Canceled', 'Skipped'].includes(this.message.status) &&
+        !this.message.type
```

Comment:
what is this? to check it is not a composition? it's better to avoid using `type` as a variable name as it is a reserved word, and here it is very unclear of what it is (I would not guess what it is)